src/app/admin/websiteDataTable/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import { GridColumnHeaderParams, GridValidRowModel } from "@mui/x-data-grid";
+import React from "react";
+
+const Header = <Data extends GridValidRowModel>({
+    colDef,
+}: GridColumnHeaderParams<Data>): React.ReactElement => {
+    return colDef.headerName ? (
+        <p
+            style={{ fontWeight: "bold" }}
+        >{`${colDef.headerName[0].toUpperCase()}${colDef.headerName.slice(1)}`}</p>
```

Comment:
just for styling if linter allows this
```suggestion
        <p style={{ fontWeight: "bold" }}>
            {`${colDef.headerName[0].toUpperCase()}${colDef.headerName.slice(1)}`}
        </p>
```

Reply:
linter allows